sdcm/prometheus.py

```diff
@@ -55,14 +54,11 @@ def start_metrics_server():
     https://github.com/prometheus/prometheus/wiki/Default-port-allocations
     Occupied port 9389 for SCT
     """
-    hostname = socket.gethostname()
-
     try:
         LOGGER.debug('Try to start prometheus API server')
         httpd = start_http_server(0)
         port = httpd.server_port
-        ip = socket.gethostbyname(hostname)
-
+        ip = get_my_ip()
```

Comment:
A private IP will be returned in GCE runner (not public), but it can't be accessed from builder.

Reply:
Yes, but there is no difference with what we currently have.

Reply:
I have created a task to address this issue - https://trello.com/c/qLbTAirF/4160-fix-getmyip-to-return-public-runner-ip-address-if-ipsshconnectionspublic